airbyte-integrations/connectors/source-amazon-ads/metadata.yaml

```diff
@@ -33,6 +33,13 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      6.0.0:
+        message: "`sponsoredDisplayReportStream` stream have updated schemas."
+        upgradeDeadline: "2024-11-01"
+        scopedImpact:
+          - scopeType: stream
+            impactedScopes:
+              - "sponsored_display_report_stream"
```

Comment:
I would also assume `SponsoredBrandsReportStream` would change based on the description of the ticket. Is there something I'm missing here?

Reply:
updated